Files/Filesystem/FilesystemOperations/Helpers/FilesystemHelpers.cs

```diff
@@ -710,22 +714,18 @@ public async Task<ReturnResult> RenameAsync(IStorageItemWithPath source, string
 
             if(source.ItemType == FilesystemItemType.File && Path.GetExtension(source.Path) != Path.GetExtension(newName))
 			{
-                var dialog = new MessageDialog("If you change a file extension, the file might become unusable." 
-                                                + Environment.NewLine +
-                                                "Are you sure you want to change it?", "Rename");
+                var renameDialogText = resourceLoader.GetString("RenameFileDialog/Text");
```

Comment:
You can use `"RenameFileDialog/Text".GetLocalized()` instead of resourceLoader.

Reply:
Done.